Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -751,7 +753,7 @@ theorem lieSpan_mono {t : Set M} (h : s ⊆ t) : lieSpan R L s ≤ lieSpan R L t
   rw [lieSpan_le]
   exact Subset.trans h subset_lieSpan
 
-theorem lieSpan_eq : lieSpan R L (N : Set M) = N :=
+theorem lieSpan_eq (N : LieSubmodule R L M) : lieSpan R L (N : Set M) = N :=
```

Comment:
What's this about?

Reply:
Makes the statement much more readable, at least to me!